docs/topics/http/urls.txt

```diff
@@ -42,7 +42,8 @@ algorithm the system follows to determine which Python code to execute:
    this is the value of the :setting:`ROOT_URLCONF` setting, but if the incoming
    ``HttpRequest`` object has an attribute called ``urlconf`` (set by
    middleware :ref:`request processing <request-middleware>`), its value
-   will be used in place of the :setting:`ROOT_URLCONF` setting.
+   will be used in place of the :setting:`ROOT_URLCONF` setting. If ``urlconf``
+   is set to ``None``, it will fall back to the value of :setting:`ROOT_URLCONF`.
```

Comment:
Don't forget a `.. versionchanged:: 1.9` annotation. (I'd probably put it at the end of the list).